src/components/RequestModal/AdvancedRequester/index.tsx

```diff
@@ -198,7 +198,7 @@ const AdvancedRequester: React.FC<AdvancedRequesterProps> = ({
     <>
       <div className="flex items-center mb-2 font-bold tracking-wider">
         <svg
-          className="w-4 h-4 mr-1"
+          className="w-6 h-6 mr-1"
```

Comment:
Why did we change the icon size here?

Reply:
Also, I prefer to use the older color scheme for the dropdowns in this component.